tutorials/algorithms/11_trotterQRTE.ipynb

```diff
@@ -0,0 +1,998 @@
+{
```

Comment:
<p>I would like to also see a bit of an explanation of how you create the TimeEvolutionProblem instance. The idea for this class is to contain all of the problem information, the hamiltonian, initial state, etc. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1465/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>